### PR TITLE
Refactor cron pages for mobile

### PR DESCRIPTION
**Pull request by AlecRust**
All fields are now in one form, with the "generate" fields detached from submit using "form=""".

Also rewrites the JS for these forms to remove jQuery and jQuery UI usage, namely [effect()](https://api.jqueryui.com/highlight-effect/) which has been removed from the custom jQuery UI build. Any manual testing of this would be appreciated.

No visual change on desktop.

screenshot

### CODE REVIEW

**Comment by jaapmarcus** [comment]
209443789-716baf98-0dd0-4b88-9eca-36ac4120fd81
209443792-44da0c05-c133-45ce-a2d3-3402be87f2a4
209443797-1f8ba4d1-d72c-450d-a2d6-946880dca4c3

**Comment by jaapmarcus** [comment]
Not related to this PR

Also "Expand" / "Collapse" main menu  press doesn't hide the "Menu" open window on the Server, Filemanger and Logout

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
See comments

**Comment by AlecRust** [comment]
> See comments

Thanks, I've fixed the worst of the issues in your screenshots.

image

**Comment by jaapmarcus** [comment]
5 times better...

File: web/js/pages/add_cron.js [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+			inputNames.forEach((inputName) => {
+				const value = fieldset.querySelector(`[name=h_${inputName}]`).value;
+				const formInput = document.querySelector(`#vstobjects input[name=v_${inputName}]`);
+
+				formInput.value = value;
+				formInput.classList.add('highlighted');
+
+				setTimeout(() => {
+					formInput.classList.remove('highlighted');
+				}, 500);
```
Comment (jaapmarcus) [inline web/js/pages/add_cron.js]: I Don't mind removing or shorting this effect. 500 ms is personally a bit to long

File: web/js/pages/edit_cron.js [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+			inputNames.forEach((inputName) => {
+				const value = fieldset.querySelector(`[name=h_${inputName}]`).value;
+				const formInput = document.querySelector(`#vstobjects input[name=v_${inputName}]`);
+
+				formInput.value = value;
+				formInput.classList.add('highlighted');
+
+				setTimeout(() => {
+					formInput.classList.remove('highlighted');
+				}, 500);
```
Comment (jaapmarcus) [inline web/js/pages/edit_cron.js]: Same for here

**Review by jaapmarcus — CHANGES_REQUESTED** [review]

File: web/js/pages/add_cron.js (hunk 1 above)
Comment (AlecRust) [reply web/js/pages/add_cron.js]: Sure, I've halved this timeout.

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**